electron/actions/app/setup.js

```diff
@@ -14,6 +17,12 @@ export default function () {
 
   createAboutWindow()
 
+  if (app.commandLine.hasSwitch(
+    'open-about-window'
+  )) {
+    aboutWindow.show()
```

Comment:
@gamersi do we still need this even after you handled it in `handleSecondInstance`?

Reply:
Maybe you'd need that when it's NOT a second instance?

Reply:
Yes